### PR TITLE
Removed mock dependency, Added mypy support

### PR DESCRIPTION
**Pull request by omri-globality**
Why?

To support latest conventions.

### CODE REVIEW

File: {{cookiecutter.repo_name}}/.gitignore [hunk 1]
```diff
@@ -102,3 +102,6 @@ build.{{ cookiecutter.project_name }}/docker-base/*
 
 # microcosm config
 {{ cookiecutter.project_name }}.conf
+
+[mypy]
```
Comment (jessemyers) [inline {{cookiecutter.repo_name}}/.gitignore]: This is the `setup.cfg` config; so it's not what you want here.

Comment (omri-globality) [reply {{cookiecutter.repo_name}}/.gitignore]: Oops, will update.

**Review by jessemyers — APPROVED** [review]

**Comment by adamhadani** [comment]
ping @omri-globality still relevant? or can delete/close this PR?

**Comment by omri-globality** [comment]
@adamhadani Closing this since it's not relevant anymore.

**Pull request closed without merging**